Lombiq.Hosting.Tenants.Maintenance/Readme.md

```diff
@@ -95,18 +95,19 @@ The following configuration options are available to set the shell request URLs:
 
 **NOTE**: The `{TenantName}` placeholder will be replaced with the actual tenant name automatically.
 
-### `Lombiq.Hosting.Tenants.Maintenance.RemoveLoginInfos`
+### `Lombiq.Hosting.Tenants.Maintenance.RemoveUsers`
 
-It's a maintenance task that removes the login infos of users from the database. It is available only for the default tenant. Useful if you have Azure AD enabled in your production environment and you want to reset staging to the production database. Then you would get "System.InvalidOperationException: Provider AzureAD is already linked for userName" error without removing the login infos.
+It's a maintenance task that removes users from the database with the given email domain. It is available only for the default tenant. Useful if you have Azure AD enabled in your production environment and you want to reset staging to the production database. Then you would get "System.InvalidOperationException: Provider AzureAD is already linked for userName" error, so deleting those users.
```

Comment:
```suggestion
It's a maintenance task that removes users from the database with the given email domain. It is available only for the default tenant. Useful if you have Azure AD enabled in your production environment and you want to reset staging to the production database. Then you would get "System.InvalidOperationException: Provider AzureAD is already linked for userName" error, so deleting those users will solve the error.
```
...or something like that, because it feels unfinished.